apis/v1alpha1/clusterprofile_types.go

```diff
@@ -71,12 +71,23 @@ type ClusterProfileStatus struct {
 	// +optional
 	Properties []Property `json:"properties,omitempty"`
 
-	// CredentialProviders is a list credential providers that can provide kubeconfig
-	// credential to connect to the cluster.
-	CredentialProviders []CredentialProvider `json:"credentialProviders,omitempty"`
+	// CredentialProviders is deprecated. Use AccessProviders instead.
+	// +optional
+	// +deprecated
+	CredentialProviders []AccessProvider `json:"credentialProviders,omitempty"`
```

Comment:
To minimise changes and avoid breaking existing users, use the old type:

```suggestion
	CredentialProviders []CredentialProvider `json:"credentialProviders,omitempty"`
```